File: atv1/python/main.py
from Login import Login
from LoginEspecial import LoginEspecial

def main():

    login = Login('eduardo', '123')

    login_especial = LoginEspecial('kauan', '132', 'primeiros numeros')

    print(login.verificalogin('carlos', '123'))
    print(login_especial.getDica())

if __name__ == "__main__":
    main()

File: atv1/java/Main.java
package atv1.java;

public class Main {

    public static void main(String[] args) {
        Login login = new Login("eduardo", "123");

        System.out.println(login.verificaLogin("carlos", "123"));

        LoginEspecial loginEspecial = new LoginEspecial("kauan", "123", "3 primeiros números");

        System.out.println(loginEspecial.getDica());
    }
    
}
